common/src/main/java/com/microsoft/identity/common/internal/platform/DevicePopManager.java

```diff
@@ -808,6 +808,42 @@ public String decrypt(@NonNull final Cipher cipher,
         throw clientException;
     }
 
+    @Override
+    public SecureHardwareState getSecureHardwareState() throws ClientException {
+        final String errCode;
+        final Exception exception;
+
+        final KeyStore.Entry keyEntry;
+        try {
+            keyEntry = mKeyStore.getEntry(mKeyAlias, null);
+            final KeyPair rsaKeyPair = getKeyPairForEntry(keyEntry);
+            return getSecureHardwareState(rsaKeyPair);
+        } catch (final KeyStoreException e) {
+            errCode = KEYSTORE_NOT_INITIALIZED;
+            exception = e;
+        } catch (final NoSuchAlgorithmException e) {
+            errCode = NO_SUCH_ALGORITHM;
+            exception = e;
+        } catch (final UnrecoverableEntryException e) {
+            errCode = INVALID_PROTECTION_PARAMS;
```

Comment:
Can we have more details on what could cause this?

Reply:
fb6577a6747c4962148455fa7978336157bd83b6